docs/content/preview/drivers-orms/csharp/postgres-npgsql.md

```diff
@@ -90,14 +87,16 @@ connStringBuilder.TrustServerCertificate = true;
 CRUD(connStringBuilder.ConnectionString);
 ```
 
-| Parameter | Description | Default |
+| Npgsql Parameter | Description | Default |
 | :---------- | :---------- | :------ |
 | sslmode  | SSL Mode | require
 | TrustServerCertificate |  Trust the server certificate configured on the YugabyteDB cluster | true
```

Comment:
Above it says "You can override this by specifying Trust Server Certificate = true" - so is true really the default? If require fails without TrustServerCertificate doesn't that mean it is false by default? Also, above it is Trust Server Certificate, here it is TrustServerCertificate